.github/workflows/main.yml

```diff
@@ -0,0 +1,43 @@
+name: "Prerender events"
+
+on:
+  push:
+    branches:
+      - master
+  schedule:
+    -   cron: "0 4 * * *"
+
+jobs:
+  backup:
+    runs-on: ubuntu-latest
+    name: Prerender
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+
+      - name: Run Prerender
```

Comment:
I think it should be ran on the branch the build was executed

Reply:
True